docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+
+services:
+  coordinator:
+    build:
+      context: ./
+      dockerfile: ./cmd/coordinator/Dockerfile
+    ports:
+      - "8080:8080"
+
+  worker:
+    build:
+      context: ./
+      dockerfile: ./cmd/worker/Dockerfile
+    ports:
+      - "8081:8081"
+    depends_on:
+      - coordinator
+    environment:
+      - COORDINATOR_HOST=coordinator
+      - COORDINATOR_PORT=8080
```

Comment:
I'm thinking, this for another pr though: Can we have an image that is a worker starter that actually creates n separate workers and we can define the amount through env variable? The port setting would be the only challenge for it, but I think it can just be done automatically inside and since we're connecting through internal docker network it will be a very useful tool to have

Reply:
sure, sounds useful